telco-ran/configuration/kube-compare-reference/required/cluster-tuning/monitoring-configuration/ReduceMonitoringFootprint.yaml

```diff
@@ -11,5 +12,31 @@ data:
       enabled: false
     telemeterClient:
       enabled: false
+    nodeExporter:
+      collectors:
+        buddyinfo: {}
+        cpufreq: {}
+        ksmd: {}
+        mountstats: {}
+        netclass: {}
+        netdev: {}
+        processes: {}
+        systemd: {}
+        tcpstat: {}
     prometheusK8s:
-       retention: 24h
+      additionalAlertmanagerConfigs:
+      - apiVersion: v2
+        bearerToken:
+          key: token
+          name: observability-alertmanager-accessor
+        scheme: https
+        staticConfigs:
+        - {{ (regexFind "alertmanager-endpoint(.*)" ((fromSecret "open-cluster-management-addon-observability" "hub-info-secret" "hub-info.yaml") | base64dec)) | replace "alertmanager-endpoint: https://" "" }}
```

Comment:
Under the kube-compare reference this should be a different syntax than in the policy. No lookup (fromSecret) can be done in the kube-compare context. Instead this should use one of the regex to allow any valid user input. eg [capturegroup inline diff function](https://github.com/openshift/kube-compare/blob/main/docs/reference-config-guide-v2.md#capturegroup-inline-diff-function)